### PR TITLE
Silk into clothing, bows, arrows, quivers and more!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Adds in 2 new bows - Pipe and Bone bow
Pipe is less good at shooting then bone, but anyone can craft it.
Ash bow is locked to people that learn it as well as any ash walker that wishes to scripe them the ways.
New arrows, Bone and harden, All need silk and wood at lest to make well the better arrows are locked to ash walkers again unless they make you the book. 
New book (rejoice!) Sand stone  tablet that teaches anyone thats not an ash walker how to craft these better bows and arrows
All arrows and bows can be sold in cargo for quite a bit of coin, but not as much as if you were to sell the mats raw as its a bow and arrow, not all that well used in the 26th century.
Silk can now be used to make white gloves, and white jumpsuits, as well as the string needed for bow and arrow crafting.
Stone tablet being rather vauleable do to being a book, in stone, like old days, about a lost art.
Ash walkers round start can only make these stone tablets, bone bow and some arrows 
Quivers! Ash walker only unless stone book once again, just holds arrows QoL for ash walkers that have more then 2 arrows.

## Why It's Good For The Game

Gives ashwalkers some range after much hard work in making the loom, cotton plants and getting the bone, also allows station based people to make rather bad bows and arrows if they were to want to make them for some unknown reason. See improvised shotgun that deals over 6x the damage, is faster to use and fits in bags.
Quivers are just bags that hold arrows so no real commit on them here other then QoL for hunter ashwalkers 

Helps give RP reasons to miners for not killing all the ash walkers and gives more range based gear to ash walkers so they are more of a threat to miners.
Allows people to use silk in other ways then just selling, more will come with this.

This also helps against the unbalancements of having anyone be able to craft the more powerfull arrows and bows without needing an ash walker to give them a book to teach them on how to craft.

## Changelog
:cl: Raiq & Linzolle
add: Bone bow - Ash walkers crafting , bone arrows - Ash walkers crafting, silk string used in bow crafting, harden arrows - Ash walkers crafting, ash walker only crafting book, basic pipe bow, and bow & arrow selling. Quivers for ash walkers as well, just to hold some arrows well out on the hunt!
/:cl:

### CODE REVIEW

File: code/game/objects/items/granters.dm [hunk 1]
```diff
@@ -490,7 +490,13 @@
 	oneuse = FALSE
 	remarks = list("Looks like these would sell much better in a plasma fire...", "Using glass bowls rather then cones?", "Mixing soda and ice-cream?", "Tall glasses with of liquids and solids...", "Just add a bit of icecream and cherry on top?")
 
-//Later content when I have free time - Trilby Date:24-Aug-2019
+/obj/item/book/granter/crafting_recipe/bone_bow //Bow crafting for non-ashwalkers
+	name = "Standstone manual on bows"
```
Comment (Linzolle) [inline code/game/objects/items/granters.dm]: Sandstone

File: code/datums/components/crafting/guncrafting.dm [hunk 2]
```diff
@@ -13,3 +13,9 @@
 	desc = "A classic rifle stock that doubles as a grip, roughly carved out of wood."
 	icon = 'icons/obj/improvised.dmi'
 	icon_state = "riflestock"
+
+/obj/item/weaponcrafting/silkstrink
```
Comment (Linzolle) [inline code/datums/components/crafting/guncrafting.dm]: change all mentions of this into "silkstring"

File: code/game/objects/items/granters.dm [hunk 3]
```diff
@@ -490,7 +490,13 @@
...
+	desc = "A standstone slab with everything you need to know for making bows and arrows just like an ashwalker would."
+	crafting_recipe_types = list(/datum/crafting_recipe/bone_arrow, /datum/crafting_recipe/bone_bow, /datum/crafting_recipe/ashen_arrow, /datum/crafting_recipe/bow_tablet)
+	icon_state = "stone_tablet"
+	oneuse = FALSE
+	remarks = list("Sticking burning arrows into the sand makes them stronger?", "Braking the bone apart to get shards, not sharpening  the bone.", "Sinew is just like rope?")
```
Comment (Linzolle) [inline code/game/objects/items/granters.dm]: Breaking, not braking. Also a double space.

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm [hunk 4]
```diff
@@ -278,6 +288,37 @@
 ///AMMO CRAFTING//
 //////////////////
 
+/datum/crafting_recipe/arrow
+	name = "Arrow"
+	result = /obj/item/ammo_casing/caseless/arrow
+	time = 40
+	reqs = list(/obj/item/stack/sheet/mineral/wood = 1,
+				 /obj/item/weaponcrafting/silkstrink = 1,
```
Comment (Linzolle) [inline code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm]: do ashwalkers even have access to silk?

Comment (Linzolle) [reply code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm]: wait nvm they have the bone arrows.

**Comment by Linzolle** [comment]
consider adding a way to craft the Quiver. Ashwalkers can get access to Leather from skinning goliaths so that can be used in recipes.

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm]: Yes, cotton into cloth, cloth into silk

File: code/modules/projectiles/projectile/reusable/arrow.dm [hunk 5]
```diff
@@ -3,4 +3,23 @@
...
+	desc = "Fire harderned arrow."
+	damage = 25
+	ammo_type = /obj/item/ammo_casing/caseless/arrow/ashen
+
+/obj/item/projectile/bullet/reusable/arrow/bone //AP for ashwalkers
+	name = "bone arrow"
+	desc = "Arrow made of bone and sinew."
+	damage = 35
+	armour_penetration = 40
+	ammo_type = /obj/item/ammo_casing/caseless/arrow/ashen
```
Comment (Linzolle) [inline code/modules/projectiles/projectile/reusable/arrow.dm]: is this intended? it'll drop the ashen arrows on hit, forcing you to remake it

File: code/modules/projectiles/ammunition/caseless/arrow.dm [hunk 6]
```diff
@@ -5,4 +5,22 @@
...
+
+/obj/item/ammo_casing/caseless/arrow/ashen
+	name = "ashen arrow"
+	desc = "Fire harderned wooden arrow."
+	icon_state = "asharrow"
+	projectile_type = /obj/item/projectile/bullet/reusable/arrow/ashen
+
+/obj/item/ammo_casing/caseless/arrow/bone
+	name = "bone arrow"
+	desc = "Arrow made of bone and sinew. The tip is sharp enough to price into a golathin plate."
```
Comment (Linzolle) [inline code/modules/projectiles/ammunition/caseless/arrow.dm]: did you mean "pierce into a goliath plate"

Comment (Trilbyspaceclone) [reply code/modules/projectiles/ammunition/caseless/arrow.dm]: Yes...

**Review by silicons — APPROVED** [review]

**Pull request merged**